enhancements/windows-containers/health-management.md

```diff
@@ -168,6 +176,10 @@ When making use of a PowerShell script, retries should be present within the scr
 is waiting for the HNS network to be created in order to create an HNS endpoint. In that specific case, the PowerShell
 script which creates the endpoint should include a retry period which waits for the needed Network to exist.
 
+Each service that has the bootstrap flag set as true must have a higher priority than all non-bootstrap services. 
```

Comment:
nit: While this probably should be the case, this doesn't necessarily need to be true